umh.docs.umh.app/static/js/requirements.js

```
@@ -1,164 +1,174 @@
+      // Function to capitalize the first letter
+      function capitalizeFirstLetter(string) {
+        if (!string) return '';
+        return string.charAt(0).toUpperCase() + string.slice(1);
+      }
+
+      // Function to format OS list into natural language
+      function formatOSList(names) {
+        if (names.length === 0) return '';
+        if (names.length === 1) return names[0];
+        if (names.length === 2) return `${names[0]} or ${names[1]}`;
+        return names.slice(0, -1).join(', ') + ' or ' + names[names.length - 1];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)